src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,327 @@
+    <label for="videoSource">Video source: </label><select id="videoSource"></select>
+  </div>
+  <input class="hidden" id="zoom" type="range" step="20">
+  <video autoplay class="hidden"></video>
+  <canvas class="hidden"></canvas>
+  <img class="hidden">
+</div>
+
+## Camera capabilities
+
+In the demo above, you'll notice a difference in dimensions between the
+`grabFrame()` and `takePhoto()` results.
+
+The `takePhoto()` method gives access to the camera's maximum resolution.
+
+`grabFrame()` just takes the next-available `VideoFrame` in the `MediaStream`
+inside the renderer process, whereas `takePhoto()` interrupts the `MediaStream`,
+reconfigures the camera, takes the photo (usually in a compressed format,
+hence the `Blob`) and then resumes the `MediaStream`. In essence, this means
+that `takePhoto()` can give access to the full still-image resolution
+capabilities of the camera. Previously, it was only possible to 'take a photo' by
+calling `drawImage()` on a `canvas` element, using a video as the source (as per the
+example [here](https://webrtc.github.io/samples/src/content/getusermedia/canvas/)).
+
+In this demo, the `canvas` dimensions are set to the resolution of the video
+stream, whereas the (natural) size of the `<img>` is the maximum resolution of
+the camera — though, of course, CSS is used to set the display size of both.
+
+The full range of possible camera resolutions for still images can be accessed
+via the `MediaSettingsRange` values for `PhotoCapabilities.imageHeight` and
+`imageWidth`. Note that the minimum and maximum width and height constraints for
+`getUserMedia()` are for video, which (as discussed) may be different from the
+camera capabilities for still images. In other words, you may not be able to
+access the full resolution capabilities of your device when saving from
+`getUserMedia()` to a canvas. The WebRTC [resolution constraint
+demo](https://webrtc.github.io/samples/src/content/getusermedia/resolution)
+shows how to set `getUserMedia()` constraints for resolution.
+
+## Anything Else?
+
+* The **Shape Detection API** works well with `ImageCapture`: call `grabFrame()`
+repeatedly to feed `ImageBitmap`s to a `FaceDetector` or `BarcodeDetector`. Find
+out more from Paul Kinlan's [blog post](https://paul.kinlan.me/face-detection/).
+
+* **Camera flash** (torch) can be accessed via
+`[FillLightMode](https://w3c.github.io/mediacapture-image/#FillLightMode)`.
+
+## Demos &amp; Code Samples
+* [Chrome Samples demo](https://github.com/GoogleChrome/samples/tree/gh-pages/image-capture)
+* [simpl.info/ic](https://simpl.info/ic)
+* [WebRTC samples](https://webrtc.github.io/samples)
+
+## Support
+* Chrome 56 on desktop and Android as an [Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md).
+* Chrome Canary on desktop and Android with **Experimental Web Platform** features enabled.
+
+## Find Out More
+* [ImageCapture spec](https://www.w3.org/TR/image-capture/)
```

Comment:
Perhaps interesting to add here the implementation status
https://github.com/w3c/mediacapture-image/blob/gh-pages/implementation-status.md